### PR TITLE
Update doc about deploying ESF using SAR AWS console

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to update the doc about the limitation of deploying ESF using SAR AWS console: one deployment per region.

### CODE REVIEW

File: docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc [hunk 1]
```diff
@@ -333,7 +335,8 @@ NOTE: To deploy the forwarder directly without using SAR, refer to <<aws-serverl
 [role="screenshot"]
 image::images/aws-serverless-forwarder-create-function.png[Create Elastic Serverless Forwarder Lambda function within SAR]
 +
-. Complete the **Application settings** according to <<aws-serverless-forwarder-define-deploy-parameters>>.
+. Complete the **Application settings** according to <<aws-serverless-forwarder-define-deploy-parameters>>. 
+Please specify the parameters even if it's already exist in the `config.yaml` file.
```
Comment (constanca-m) [inline docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc:339]:
```suggestion
Please specify the parameters even if they already exist in the `config.yaml` file.
```

**Review by constanca-m — DISMISSED** [review]

File: docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc [hunk 2]
```diff
@@ -333,7 +335,8 @@ NOTE: To deploy the forwarder directly without using SAR, refer to <<aws-serverl
 [role="screenshot"]
 image::images/aws-serverless-forwarder-create-function.png[Create Elastic Serverless Forwarder Lambda function within SAR]
 +
-. Complete the **Application settings** according to <<aws-serverless-forwarder-define-deploy-parameters>>.
+. Complete the **Application settings** according to <<aws-serverless-forwarder-define-deploy-parameters>>. 
+Please specify the parameters even if they already exist in the `config.yaml` file.
```
Comment (tetianakravchenko) [inline docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc:339]:
```suggestion
You must specify the parameters even if they already exist in the `config.yaml` file.
```
should be also mentioned explicitly that there must be defined at least one of the parameters: `ElasticServerlessForwarderSQSEvents`/`ElasticServerlessForwarderS3SQSEvents`/`ElasticServerlessForwarderKinesisEvents`/ `ElasticServerlessForwarderCloudWatchLogsEvents`, depending on the used input type ?

**Comment by zmoog** [comment]
We set mypy == 1.10.1 and types-requests<2.31.0.7 as a workaround, creating individual issues to understand the problems better.

Here is the mypy one at https://github.com/elastic/elastic-serverless-forwarder/issues/772

**Review by constanca-m — APPROVED** [review]

**Comment by kaiyan-sheng** [comment]
@zmoog Thanks for your help with the workaround!!

**Pull request merged**